### ISSUE TITLE
2024.8 - Change `service-call` to `perform-action` and Fix Config UI

### ISSUE THREAD

**Issue by Nerwyn** [Nerwyn/service-call-tile-feature#43]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
2024.8

**Last working release (if known):**
2024.7

**Browser and Operating System:**

**Description of problem:**
As of HA 2024.8, the `service-call` action has been renamed to `perform-action`, with the field `service` changed to `perform_action` to match. While old configs will continue to work the configuration UI does not. Both the base action logic and UI should be updated to use the new fields. I should also check if `hass.callService` is still the appropriate frontend function to call.

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Your Full Card Config**

```yaml
show_name: true
show_icon: true
type: button
tap_action:
  action: perform-action # Formerly call-service
  perform_action: light.toggle # Formerly service
  target:
    entity_id: light.desk_lights
entity: light.desk_lights
```

**Additional information:**
![image](https://github.com/user-attachments/assets/b571e4e2-ea64-4fd1-aae2-d9c7f21a4222)

**Comment by Nerwyn** [comment]
The selector for different features example should be updated to work better with sections forced grid row height.

**Comment by Nerwyn** [comment]
Also need to fix `.toString()` on option value comparison to handle undefined options and slider thumb styles.

**Comment by Nerwyn** [comment]
Fixed in [4.0.7](https://github.com/Nerwyn/service-call-tile-feature/releases/tag/4.0.7)
